src/signpdf.js

```diff
@@ -134,21 +134,6 @@ export class SignPdf {
             key: privateKey,
             certificate,
             digestAlgorithm: forge.pki.oids.sha256,
-            authenticatedAttributes: [
```

Comment:
Again, I want to know how this affects signatures. Other than making them valid. I know it does that. But why? How? Need to know more.